src/components/EditableField/EditableField.Input.jsx

```diff
@@ -81,6 +81,7 @@ export class EditableFieldInput extends React.Component {
 
   inputRef
   inputWrapperRef = React.createRef()
+  _isMounted = false
```

Comment:
is it a cleanup change from you, or it's an overwrite to `main` that we don't want?

Reply:
From me, got rid of a memory leak warning on EditableField.Input that was bothering me and included it here.